charts/knative-serving-core/crds/autoscaling-metrics.yaml

```diff
@@ -19,12 +19,11 @@ kind: CustomResourceDefinition
 metadata:
   name: metrics.autoscaling.internal.knative.dev
   labels:
-    helm.sh/chart: knative-serving-core
-    app.kubernetes.io/version: "1.3.2"
-    serving.knative.dev/release: "v1.3.2"
```

Comment:
This label us used by pod monitors. See https://github.com/caraml-dev/helm-charts/blob/main/charts/knative-serving-core/values.yaml#L205. 
Can we not remove this from others as well.

Reply:
FYI, I think we should not use that labels anymore as it has been deprecated by knative (https://github.com/knative/serving/issues/12215). To avoid any issue with monitoring I will re-include the labels in this PR, but please plan to deprecate it accordingly.

Reply:
Thanks, Will create a task to migrate them. 
cc @mbruner

Reply:
Reverted the labels change.